api/DocumentOrShadowRoot.json

```diff
@@ -52,28 +52,28 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/activeElement",
           "support": {
             "chrome": {
-              "version_added": "53"
+              "version_added": "1"
```

Comment:
So here's the problem with mixins. `ShadowRoot` didn't exist in Chrome 1. But `document.activeElement` was there. So what do we do?

Reply:
Based upon our data, `ShadowRoot` is also not around in Chrome 53 either (though results from the `update-bcd` script indicate that it is).  It appears that was my fault (#3403); I'll open up a PR to fix the Chromium version data.

I think that overall, this is probably fine to set versions for these features earlier than when `ShadowRoot` was around.  I remember that when we were talking about how to mixins before, we had talked about basing support on when the method/attribute was supported on all of the interfaces that include it (disregarding versions when an interface is not available) -- this is basically how that data would look.

Reply:
In https://github.com/mdn/browser-compat-data/issues/472#issuecomment-696628301 I list two options, and this would be the first ("any") option. @saschanaz also preferred that.

@ddbeck @Elchi3 do you think we should go ahead an update existing mixin data like this, or should we leave it alone and delete mixins entirely?

Reply:
1. If you're going to update mixin data, then the "any" option seems to be the only sustainable approach.
2. We should probably come up with a plan to de-mixin the data. But there's… a lot to unpack there, so I imagine it will take some time.
3. I suspect mixins will endure long enough that it won't be a waste of time to update the mixin data and I would welcome updates to it. But I would understand if, all other things being equal, you prioritized updating data for concrete interfaces ahead of mixins.

Reply:
Thanks @ddbeck! Let's go with the "any" option for now then.